de.metas.acct.base/src/main/sql/postgresql/ddl/functions/AccountSheetReport.sql

```diff
@@ -0,0 +1,238 @@
+DROP FUNCTION IF EXISTS AccountSheetReport(p_dateFrom date, p_dateTo date, p_c_acctschema_id NUMERIC, p_ad_org_id numeric, p_account_id NUMERIC, p_c_activity_id numeric, p_c_project_id numeric);
+
+/*
+- DateFrom/To - mandatory
+- C_AcctSchema_ID - mandatory
+- AD_Org_ID - mandatory
+- Account_ID - optional
+- C_Activity_ID - optional
+- C_Project_ID - optional
+*/
+CREATE OR REPLACE FUNCTION AccountSheetReport(p_dateFrom date,
+                                              p_dateTo date,
+                                              p_c_acctschema_id NUMERIC,
+                                              p_ad_org_id numeric,
+                                              p_account_id NUMERIC=NULL,
+                                              p_c_activity_id numeric=NULL,
+                                              p_c_project_id numeric=NULL)
+    RETURNS table
+            (
+                dateacct         timestamp WITHOUT TIME ZONE,
+                documentno       text,
+                description      text,
+                amtacctdr        numeric,
+                amtacctcr        numeric,
+                account_id       numeric,
+                fact_acct_id     numeric,
+                c_doctype_id     numeric,
```

Comment:
export doc type name instead of c_doctype_id
same of the other ID columns

Reply:
done